services/order.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic()
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None) -> Order:
+
+    user = get_user_model().objects.get(username=username)
```

Comment:
Consider handling exceptions when retrieving the user by username. If the user does not exist, `get()` will raise a `DoesNotExist` exception, which should be caught and handled appropriately.